app/src/vanilla/java/cm/aptoide/pt/app/view/EditorialFragment.java

```diff
@@ -504,4 +658,19 @@ private void handleStatusBar(boolean collapseState) {
       }
     }
   }
+
+  private boolean isScrollDown(int oldY, int newY) {
+    return newY > oldY;
+  }
+
+  private boolean isItemShown() {
+    return adapter.isItemShown(screenHeight, screenWidth);
+  }
+
+  private View getPlaceHolder() {
+    if (appCardPlaceHolder == null) {
+      return adapter.getPlaceHolder();
```

Comment:
For same reason as the above, i don't agree with saving the placeholder viewholder on the adapter